lib/ansible/modules/windows/win_webpicmd.py

```diff
@@ -33,11 +33,11 @@
 short_description: Installs packages using Web Platform Installer command-line
 description:
     - Installs packages using Web Platform Installer command-line
-      (http://www.iis.net/learn/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release).
-    - Must be installed and present in PATH (see win_chocolatey module; 'webpicmd' is the package name, and you must install 'lessmsi' first too)
-    - Install IIS first (see win_feature module)
+      (U(http://www.iis.net/learn/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release)).
```

Comment:
This is still a bit confusing.  How about putting the things that need doing first first in the list?  I have not used so I'm guessing here, but how about something like this?

Before using this module, ensure that you have
Installed IIS ( see M(win_feature))
Installed 'lessmsi' and 'webpicmd'.  Both 'lessmsi' and 'webpicmd' can be installed using M(win_chocolatey).
Alternatively (U(http://www.iis.net/learn/install/web-platform-installer/web-platform-installer-v4-command-line-webpicmdexe-rtw-release)).